bvm/ballerina-core/src/main/java/org/ballerinalang/model/values/BRefValueArray.java

```diff
@@ -76,7 +77,12 @@ public BValue copy() {
     
     @Override
     public String stringValue() {
-        StringJoiner sj = new StringJoiner(", ", "[", "]");
+        StringJoiner sj;
+        if (arrayType.getTag() == TypeTags.TUPLE_TAG) {
```

Comment:
Can we do arrayType != null check here before getting tag. 

Following test case is failing. I suspect that, grpc "testStructArrayInput" native action doesn't set correct return value's type.

testStructArrayInputClient(org.ballerinalang.test.service.grpc.sample.UnaryBlockingArrayValueTestCase)  Time elapsed: 0.829 sec  <<< FAILURE!
org.ballerinalang.util.exceptions.BLangRuntimeException: 
error: error
	at ballerina.io:println(<native>)

	at org.ballerinalang.test.service.grpc.sample.UnaryBlockingArrayValueTestCase.testStructArrayInputClient(UnaryBlockingArrayValueTestCase.java:178)